document/CoreFormat.mdk

```diff
@@ -1445,6 +1453,36 @@ or if the tool cannot or chooses not to determine whether the result occurred at
 or if the tool examines source code written in a language that lacks such a construct, the `suppressionStates` array
 shall not include the value `"suppressedInSource"`.
 
+### `baselineState` property { #result-baselineState }
+
+If the `baselineId` property (&sect;[#run-baselineId]) of the current `run` object (&sect;[#run]) is present,
+then the `result` object may contain a property named `baselineState` whose value is a string that shall specify
+the state of this `result` with respect to the run specified by `baselineId`.
+
+If the `run.baselineId` property is absent, the `baselineState` property shall be absent.
+
+This property shall have one of the following values, with the specified meanings:
+
+* `"new"`: This result was detected in the current run but was not detected in the run specified by `run.baselineId`.
+
+* `"existing"`: This result was detected both in the current run and in the run specified by `run.baselineId`.
+
+* `"absent"`: This result was detected in the run specified by `run.baselineId` but was not detected in the current run.
+
+If the `run.baselineId` property is present but the `baselineState` property is absent, the `baselineState` property
+shall be considered to have the value `"new"`.
+
+~ IsoNote
+The purpose of the `baselineState` property is to allow result management systems to keep track of how many
+new results are introduced in each run, and how many have been addressed.
```

Comment:
no, we don't address the issue of what issues are addressed, that's a future item for something like 'trackingState' or 'resolutionState'. baselineState is strictly restricted to providing information about the intersection of two result sets, into three groups, present exclusively in one, present exclusively in the other, or present in both.